aap-14811 Added modules for section 4.5 (#812)
ansible/aap-docs: downstream/modules/aap-hardening/con-aap-patch-automation.adoc

// Module included in the following assemblies:
// downstream/assemblies/assembly-aap-security-use-cases.adoc

[id="con-aap-patch-automation_{context}"]

= Patch automation with {PlatformNameShort}

[role="_abstract"]

> Software patching is a fundamental activity of security and IT operations teams everywhere.  Keeping patches up to date is critical to remediating software vulnerabilities and meeting compliance requirements, but patching systems manually at scale can be a time-consuming and error-prone process.  Organizations should put thought into patch management strategies that meet their security, compliance, and business objectives, in order to prioritize the types of patches to apply (known exploits, critical or important vulnerabilities, optimizations, routine updates, new features, and so on) against the IT assets available across the enterprise.

Reviewer: ```suggestion
Software patching is a fundamental activity of security and IT operations teams everywhere.
Keeping patches up to date is critical to remediating software vulnerabilities and meeting compliance requirements, but patching systems manually at scale can be a time-consuming and error-prone process.
Organizations should put thought into patch management strategies that meet their security, compliance, and business objectives, in order to prioritize the types of patches to apply (known exploits, critical or important vulnerabilities, optimizations, routine updates, new features, and so on) against the IT assets available across the enterprise.
```

> Once policies and priorities have been defined and a patching plan is established, the manual tasks involved in patch management can be automated using the {PlatformNameShort} to improve patch deployment speed and accuracy, reduce human error, and limit downtime.

Reviewer: ```suggestion
Once you have defined policies and priorities and established a patching plan, you can automate the manual patch management tasks using the {PlatformNameShort}.  Automating the tasks improves patch deployment speed and accuracy, reduces human error, and limits downtime.
```
